### PR TITLE
Update insider-risk-management-configure.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Office 365 E3 + Enterprise Mobility and Security E3 + the Microsoft 365 E5 Compliance add-on reference document  https://www.microsoft.com/en-gb/microsoft-365/business/e5-compliance?activetab=pivot%3aoverviewtab

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3972

File: microsoft-365/compliance/insider-risk-management-configure.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Before you get started with insider risk management, you should confirm your [Mi
 
 - Microsoft 365 E5 subscription (paid or trial version)
 - Microsoft 365 E3 subscription + the Microsoft 365 E5 Compliance add-on
+- Office 365 E3 + Enterprise Mobility and Security E3 + the Microsoft 365 E5 Compliance add-on
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-configure.md]:
```suggestion
- Office 365 E3 subscription + Enterprise Mobility and Security E3 + the Microsoft 365 E5 Compliance add-on
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@yogkumgit  and @msbemba - confirming with PG and Marketing

**Comment by robmazz** [comment]
@yogkumgit and @msbemba - marketing folks have reached consensus and will add, but in different place on the list. Closing this and will merge separately.

**Pull request closed without merging**